modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectDefinitionExternalReferenceCodeException.java

```diff
@@ -22,24 +22,46 @@ public ForbiddenUnmodifiableSystemObjectDefinitionExternalReferenceCode(
 			String externalReferenceCode) {
 
 			super(
+				StringBundler.concat(
+					"Forbidden unmodifiable system object definition external ",
+					"reference code ", externalReferenceCode),
 				StringBundler.concat(
 					"Forbidden unmodifiable system object definition external ",
 					"reference code ", externalReferenceCode));
```

Comment:
We need to create the language key, something like this:
`forbidden-unmodifiable-system-object-definition-external-reference-code=Forbidden unmodifiable system object definition external reference code.`

And follow the same pattern the comment above, please